04week/loop.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// Create a new file called loops.js in the /04week folder of your workbook.
+// Complete each of the following exercises.
+// for loop
+
+// Use a for loop to console.log each item in the array carsInReverse.
+const carsInReverse = ['Ford', 'Chevy', 'Tesla', 'Nissan', 'Toyota'];
+for (i = 0; i < carsInReverse.length; i++) {
+  console.log(carsInReverse[i]);
+}
+
+// Create an object (an array with keys and values) called persons with the following data:
+// firstName: "Jane"
+// lastName: "Doe"
+// birthDate: "Jan 5, 1925"
+// gender: "female"
+const persons = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+};
+
+// Use a for...in loop to console.log each key.
+for (const keys in persons) {
```

Comment:
Here, you're logging all keys (in an array) for every key, so `[ 'firstName', 'lastName', 'birthDate', 'gender' ]` gets logged four times. For the sake of the project, change `keys` to `key` (we don't have to, but just for clarity's sake since each key is only one key, not multiple), and also, just `console.log(key)`, since `Object.keys(...)` is an array not a single value.